www/site/content/docs/reference/config.md

```diff
@@ -0,0 +1,263 @@
+* Notes:
+  * Only set this if you have access to Chef's internal QA environment - otherwise the telemetry data will not be successfully captured by Chef.
+
+### [log]
+
+Control logging level and location.
+
+#### Example
+
+```toml
+[log]
+level="debug"
+location="C:\Users\username\chef-workstation.log"
+```
+
+#### Option: level
+
+* Description: determines what kind of messages are logged from locally-run Chef Workstation commands to the to the local log file.
+* Default: `"warn"`
+* Valid values: `"debug"`, `"warn"`, `"info"`, `"error"`, `"fatal"`
+* Used by: `chef-run`
+
+#### Option: location
+
+* Description: The location of the local Chef Workstation log file.
+* Default: `"$USERHOME/.chef-workstation/logs/default.log"`
+* Valid values: Value must be a valid, writable file path.
+* Used by: `chef-run`
+
+### [cache]
+
+Configure caching options.
+
+#### Example
+
+```tom
+[cache]
+path="/home/users/username/.cache/chef-workstation"
+```
+
+#### Option: path
+
+* Description: The base path used to store cached cookbooks and downloads.
+* Default: `$USERHOME/.chef-workstation/cache`
+* Valid values: This must reference a valid, writable directory.
+* Used by: `chef-run`
+
+### [connection]
+
+Control default connection behaviors.
+
+#### Example
+
+```toml
+[connection]
+default_protocol="winrm"
+default_user="username"
+```
+
+#### Option: default_protocol
+
+* Description: Default protocol for connecting to target hosts.
+* Default: `"ssh"`
+* Valid values: `"ssh"`, `"winrm"`
+* CLI flag: `--protocol PROTO`
+* Used by: `chef-run`
+
+#### Option: default_user
+
+* Description: Default username for target host authentication
+* Default: `root` (Linux),  `administrator` (Windows)
+* Valid values: A username that exists on the target hosts.
+* CLI flag: `--user USERNAME`
+* Used by: `chef-run`
+
+### [connection.winrm]
+
+Control connection behaviors for the WinRM protocol.
+
+#### Example
+
+```tom
+[connection.winrm]
+ssl=true
+ssl_verify=false
+```
+
+#### Option: ssl
+
+* Description: Enable SSL for WinRM session encryption
+* Default: `false`
+* Valid values: `true`, `false`
+* CLI flag: `--[no]-ssl`
+* Used by: `chef-run`
+
+#### Option: ssl_verify
+
+* Description:
+* Default: `true`
+* Valid values: `true`, `false`
+* CLI flag: --ssl-[no]-verify
+* Used by: `chef-run`
+* Notes: Intended for use in testing environments that use self-signed certificates on Windows nodes.
+
+### [chef]
+
+Configure remote Chef running on instances.
+
+#### Example
+
+```toml
+[chef]
+trusted_certs_dir="/home/username/mytrustedcerts"
+cookbook_repo_paths = [
+  "/home/username/cookbooks",
+  "/var/chef/cookbooks"
+]
+```
+
+#### Option: trusted_certs_dir
+
+* Description: Describes where to find Chef's trusted certificates. Used to ensure trusted certs are provided to the `chef-client` run on target nodes.
+* Default:  Look first for `.chef/config.rb` and use that value if provided; otherwise `"/opt/chef-workstation/embedded/ssl/certs/"`
```

Comment:
`... and use that value if provided` -> `... and use the value from there if provided`

Reply:
Well, looking at https://docs.chef.io/config_rb.html it doesn't appear that is even a documented config value...

Reply:
Also - should the path be `~/.chef/config.rb`?

Reply:
It's relative to current location with fallback to ~ , isn't it?

Reply:
I think you can find the doc for it in 'knife.rb' docs. It's a bit of a mess.